map_view/set-up-location-driven-geotriggers/src/main/resources/set_up_location_driven_geotriggers/style.css

```diff
@@ -0,0 +1,26 @@
+VBox {
+    -fx-background-color: rgba(0, 0, 0, 0.6);
+    -fx-padding: 20;
+    -fx-spacing: 5;
+    -fx-alignment: top-left;
+}
+
+.panel-region .label {
+  -fx-text-fill: white;
+}
+
+.garden-section-content {
+  -fx-font-size: 14;
+}
+
+.poi-detail {
+  -fx-background-color: #ac901e;
+  -fx-background-radius: 75px;
+  -fx-font-size: 14;
```

Comment:
```suggestion
  -fx-font-size: 14;
  -fx-padding: 10px;
```

This will tidy up the POI badges - only thing is I'm not sure if the sample viewer will ignore it / if you've worked on the css side of things there yet. Might be worth a check on the styling side of things...
![image](https://user-images.githubusercontent.com/56556822/130475869-ba5c20f0-de52-4d1b-bf6f-23a504baf124.png)

Reply:
Edit: I just checked it in the SV and looks good!

Reply:
that looks great, thanks!